package.json

```diff
@@ -16,6 +16,7 @@
     "clean:install": "pnpm clean:node_modules && pnpm install --frozen-lockfile",
     "build": "pnpm clean:bundle && turbo ready && turbo build",
     "build:firefox": "pnpm clean:bundle && turbo ready && cross-env __FIREFOX__=true turbo build",
+    "build:eslint": "tsc --build",
```

Comment:
```suggestion
    "build:eslint": "tsc -p eslint.json",
```

IMO this should looks like this, because `tsconfig.json` isn't necessary,
but include .ts config inside `eslint.json`

IMO 2 configs for that simple task is overkill, becasue you won't have other files in root, like we have in packages

Reply:
It was initially only `tsconfig.json`, I then added the `@extension/tsconfig/eslint.json` with the latest changes to keep all tsconfigs in one place.

> IMO 2 configs for that simple task is overkill, becasue you won't have other files in root, like we have in packages

What do you suggest?

Reply:
@plbstl Your current solution is good for now.
i'll refactor it a little, if i'll be able, on my branch.

Good work <3

Reply:
It works the same without this flag, in other packages we doesn't use it.
![image](https://github.com/user-attachments/assets/db354f53-4482-42b0-9ea0-5886e8c1b54c)

It's really necessary?

Reply:
Not necessarily, for this use case. See https://www.typescriptlang.org/docs/handbook/project-references.html#caveats-for-project-references

Reply:
@plbstl Ok, let's leave it here, it's seems nice option.
I'll use it for all `tsc` :)